hooks/useGetFolders.ts

```diff
@@ -0,0 +1,8 @@
+import useFetch from "hooks/useFetch";
+import type { Folder } from "types";
+
+export type GetFolderResponse = Folder;
+
+export const useGetFolders = (userId: number) => {
+  return useFetch(`${process.env.NEXT_PUBLIC_API}/users/${userId}/folders`);
+};
```

Comment:
이런식으로 타입을 잡아줄 수 있도록 useFetch가 변경되어도 좋을 것 같구요!
```suggestion
export const useGetFolders = (userId: number) => {
  return useFetch<GetFolderResponse>(`${process.env.NEXT_PUBLIC_API}/users/${userId}/folders`);
};
```

혹은 이렇게 타입을 잡아줘도 괜찮을 것 같아요!
```suggestion
export const useGetFolders = (userId: number):GetFolderResponse => {
  return useFetch(`${process.env.NEXT_PUBLIC_API}/users/${userId}/folders`);
};
```